gensim/corpora/dictionary.py

```diff
@@ -36,6 +36,8 @@ class Dictionary(utils.SaveLoad, Mapping):
         token -> tokenId.
     id2token : dict of (int, str)
         Reverse mapping for token2id, initialized in a lazy manner to save memory (not created until needed).
+    cfs : dict of (int, int)
```

Comment:
What are the memory implications of this change?

Reply:
Instead of storing V integers in the `dfs` attribute, where V is the vocabulary size, we are now storing 2V integers in the `dfs` and `cfs` attributes. [Empirically](https://en.wikipedia.org/wiki/Heaps%27_law), V is the square root of the corpus size.

Reply:
How much is this extra `dict` (in MBs of RAM), for let's say 100k vocab?

Reply:
Actually, never mind. I thought it's storing string keys, but it's actually storing ints, I'm blind. Probably no issue then 👍